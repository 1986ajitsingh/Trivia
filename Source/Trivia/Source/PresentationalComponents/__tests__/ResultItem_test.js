import renderer from 'react-test-renderer';
import React from 'react';
import ModuleUnderTest from '../ResultItem';

describe('renders correctly', () => {
  /* eslint-disable no-undef */
  test('normal', () => {
    /* eslint-enable no-undef */
    // Arrange/ Act
    const tree = renderer.create(<ModuleUnderTest />).toJSON();

    // Assert
    expect(tree).toMatchSnapshot();
  });
});
