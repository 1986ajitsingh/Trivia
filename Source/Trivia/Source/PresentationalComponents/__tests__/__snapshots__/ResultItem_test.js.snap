// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly normal 1`] = `
<View
  style={
    Object {
      "alignItems": "flex-start",
      "flex": 1,
      "flexDirection": "row",
      "justifyContent": "space-between",
      "padding": 10,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#4B0082",
        "fontSize": 40,
        "marginRight": 10,
        "textAlign": "center",
      }
    }
  >
    -
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#4B0082",
        "fontSize": 20,
        "marginRight": 10,
      }
    }
  >
    No question input given
  </Text>
</View>
`;
